### ISSUE TITLE
Highlighting multiple lines for SQL snippet fails to bring up save as snippet option

### ISSUE THREAD

**Issue by jeff-bruemmer** [metabase/metabase#12735]
**Describe the bug**
When trying to create a SQL snippet, if you highlight multiple lines then right-click on the highlighted selection to save as snippet, you often get the system menu, and not the expected Metabase menu to **Run selection** or **Save as snippet**. The behavior is inconsistent. Highlights of single lines appear not to have this issue.

**Logs**
No relevant logs from Javascript console or server.

**To Reproduce**
1. Go to SQL Editor and write some sql:

```sql
select *
from ORDERS
left join PRODUCTS
on PRODUCT_ID = PRODUCT_ID
```
2. Highlight multiple lines, eg. lines 2-4. 
3. Right-click to bring up Metabase dropdown menu.
4. (Usually) get system menu instead.

**Expected behavior**
I should be able to highlight multiple lines of SQL code and right-click on the selection to save as snippet.

**Screenshots**
![system menu instead of metbase menu](https://user-images.githubusercontent.com/17343328/84960174-d0fc8280-b0b5-11ea-8b03-a785b3a11cf8.png)

**Information about your Metabase Installation:**
```
{
  "browser-info": {
    "language": "en-US",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.7+10",
    "java.vendor": "AdoptOpenJDK",
    "java.vendor.url": "https://adoptopenjdk.net/",
    "java.version": "11.0.7",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.7+10",
    "os.name": "Linux",
    "os.version": "4.14.181-108.257.amzn1.x86_64",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "googleanalytics",
      "mysql",
      "druid",
      "postgres",
      "bigquery",
      "redshift",
      "mongo",
      "h2"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "9.6.11"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.8"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2020-06-17",
      "tag": "v0.36.0-snapshot",
      "branch": "master",
      "hash": "69aa33f"
    },
    "settings": {
      "report-timezone": "US/Pacific"
    }
  }
}
```
**Severity**
Annoying and unexpected.

**Comment by paulrosenzweig** [comment]
Thanks for the find @jeff-bruemmer! I added the fix to #12723

**Comment by paulrosenzweig** [comment]
☝️ I removed the milestone. I think that's used to generate release notes. I figure we don't need to mention this since it was fixed before snippets were released.
